frontend/src/pages/UpdateDecision/UpdateDecision.jsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+import UpdateCreateDecisionFormImpacted from "../../components/UpdateCreateDecisionForm/UpdateCreateDecisionFormImpacted";
+import UpdateCreateDecisionFormExperts from "../../components/UpdateCreateDecisionForm/UpdateCreateDecisionFormExperts";
+import UpdateCreateDecisionFormContent from "../../components/UpdateCreateDecisionForm/UpdateCreateDecisionFormContent";
+import "./UpdateDecision.scss";
+import { useDecisionContext } from "../../contexts/decisionContext";
+
+function UpdateDecision() {
+  const { decisionId } = useDecisionContext();
+  const { decisions, setDecisions } = useDecisionContext();
+  useEffect(() => {
+    fetch(`${import.meta.env.VITE_BACKEND_URL}/api/decision/${decisionId}`)
+      .then((response) => response.json())
+      .then((data) => setDecisions(data))
+      .catch((error) => console.error(error));
+  }, []);
+
+  const handleUpdateDecision = () => {
+    const apiEndpoint = `${
+      import.meta.env.VITE_BACKEND_URL
+    }/api/decision/update`;
+
+    const updatedData = {
+      paragraph_details: decisions.paragraph_details,
+      paragraph_impact: decisions.paragraph_impact,
+      paragraph_benefits: decisions.paragraph_benefits,
+      paragraph_risks: decisions.paragraph_risks,
+      paragraph_first_decision: decisions.paragraph_first_decision,
+      paragraph_finale_decision: decisions.paragraph_finale_decision,
+      decision_id: decisionId,
+    };
+
+    const updatedDatas = {
```

Comment:
Why not put this information in a useState?

Reply:
même réponse qu'a fantine au dessus